starter_code/script.js

```diff
@@ -3,13 +3,29 @@ var aiPoint = 0;
 
 // This function returns the selection of the computer
 function getAISelection() {
-    //TODO: randomly choose between 'rock', 'paper', or 'scissors'
+    var options = ["rock","scissors","paper"];
+    var random = Math.floor(Math.random() * (0 - 3 + 1)) + 0;
+    return options[random];
```

Comment:
This looks promising. just make sure to review the number you assign into `random`, it's a little off. (btw, you can have negative nums in JS too like `-3`